.vscode/tasks.json

```diff
@@ -0,0 +1,14 @@
+{
+  "version": "2.0.0",
+  "command": "yarn",
+  "isShellCommand": true,
+  "showOutput": "always",
+  "suppressTaskName": true,
+  "tasks": [
+    {
+      "taskName": "build",
+      "args": ["build:server"],
+      "isBuildCommand": true
+    }
+  ]
+}
```

Comment:
We should not commit .vscode files in git. It very specific.

Reply:
Yes, I have checked in so that everyone can get this code and can debug